### DISCUSSION TITLE
How to get an overview of which repositories consume the most GitHub Actions?

### DISCUSSION THREAD

**Discussion by multani** [community/community#38426] [Actions]
Hi :wave:

I used too many GitHub Actions in October and went over the free quota of 1000 minutes :man_facepalming: 

I had a runaway job that timed-out after 6 hours so this one didn't really help and was easy to fix, but generally speaking: how can I find which repositories / actions were running for the longest or were using the most consumable?

I'm really willing to decrease my amount of GitHub Actions running, but I couldn't find a way to analyze what was using the most of my free quota allowance :(

Thanks for your help!

**Comment by ossanna16** [comment]
Hi there @multani and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our official introduction thread
To connect with other new users, check out this Discussion to Meet Your GitHub Community Team, further your GitHub knowledge by working through our GitHub Skill 🆙 exercises, and participate in our weekly Fun Friday chats
